From db8153bd5f476114f6a1b7ac40357c8bc39331ef Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Dan=20Hor=C3=A1k?= <dan@danny.cz>
Date: Wed, 29 Jun 2011 15:08:27 +0200
Subject: [PATCH 5/7] test subdir requires xorg to be built

Something changed between 20110418 and 20110510 and the test subdir is now
processed. But it needs some libraries that are built as part of xorg driver.
---
 Makefile.am |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index cea140b..72f3d74 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -39,8 +39,7 @@ SUBDIRS = \
 	$(GLX_DIR) \
 	exa \
 	config \
-	hw \
-	test
+	hw
 
 if XORG
 aclocaldir = $(datadir)/aclocal
@@ -48,6 +47,8 @@ aclocal_DATA = xorg-server.m4
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = xorg-server.pc
+
+SUBDIRS += test
 endif
 
 EXTRA_DIST = xorg-server.pc.in xorg-server.m4 autogen.sh
-- 
1.7.10.1

